wheel/generate_type_stubs.py

```diff
@@ -305,6 +305,27 @@ def confirm_not_included_already_hashed(
     proof: bytes,
 ) -> bool: ...
 
+def validate_clvm_and_signature(
+    new_spend: SpendBundle,
+    max_cost: int,
+    constants: ConsensusConstants,
+    peak_height: int,
+    cache: Option[BLSCache],
+) -> Tuple[SpendBundleConditions, List[Tuple[SpendBundleConditions, List[Tuple[Bytes32, bytes]]]], float]: ...
+
+def get_name_puzzle_conditions(
+    spend_bundle: SpendBundle,
+    max_cost: int,
+    constants: ConsensusConstants,
+    height: int,
+) -> SpendBundleConditions: ...
```

Comment:
It doesn't look like this function is defined in this PR. Did you mean `get_conditions_from_spendbundle()`?
It would be good to understand why this wasn't caught by CI as well. Perhaps there's no python test for this function, or we don't run `mypy`

Reply:
Fixed the immediate problem, but I agree the testing for the pyi file is lacking

Reply:
Manually generating this file is I hope only a temporary solution

Pyo3 is working on automatic type stub generation (though I don't know how we'll do our custom integer types)

Reply:
we do run `mypy`, so I would have expected this to be caught if there was a test attempting to use this function (under either this name or the true name)